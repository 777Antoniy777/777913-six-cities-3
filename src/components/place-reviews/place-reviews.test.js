import React from 'react';
import renderer from "react-test-renderer";
import PlaceReviews from './place-reviews';

// set mocha data
const reviews = [
  {
    id: 1,
    body: `text`,
    rating: 5,
    name: `name`,
    date: `date`,
  },
];

it(`render PlaceReviews`, () => {
  const tree = renderer.create(
      <PlaceReviews
        reviews={reviews}
      />)
      .toJSON();

  expect(tree).toMatchSnapshot();
});
