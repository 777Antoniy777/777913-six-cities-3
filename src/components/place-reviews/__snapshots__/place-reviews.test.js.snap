// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render PlaceReviews 1`] = `
<ul
  className="reviews__list"
/>
`;
